.github/workflows/test.yml

```diff
@@ -68,3 +71,26 @@ jobs:
       run: |
         cd e2e
         python -m pytest .
+
+    - name: Rust library tests
+      run: |
+        cd rust/rsmgp-sys
+        cargo fmt -- --check
+        cargo test
+    - name: Install Cargo tarpaulin # Unable to run https://github.com/actions-rs/tarpaulin from subdirectory.
+      run: |
+          cargo install cargo-tarpaulin
+    - name: Run code coverage
+      run: |
+        cd rust/rsmgp-sys
+        RUST_BACKTRACE=1 cargo tarpaulin --out Html --ignore-tests
+    - name: Archive code coverage results
+      uses: actions/upload-artifact@v1
+      with:
+        name: rsmgp-sys-coverage-report
+        path: rust/rsmgp-sys/tarpaulin-report.html
```

Comment:
Why would you need report in CI?

Reply:
CI should usually generate the coverage report (because that's not a required operation like, e.g., running unit tests). But here, the tarpaulin installation takes a couple of minutes. That's why I'm considering removing all that from the CI for now?

Reply:
It makes sense to have these coverage reports if you use them, but if there is no automatization usage, then there is no reason to have them in Github workflow, but rather run it locally.

Reply:
Ok, will remove the uploading of the report file :smile: